2.0.5-snapshot  getting runtime error: cannot find a NameResolver

Hello,
I am using this java-client from a non-android backend java app (regular backend app with JDK 11).
Initially, I used released 2.0.4 java client.

Now I am troubleshooting some intermittent 'socket may not be closed issues' under high load with DefaultClient, and decided to upgrade to master (2.0.5-shapsot) of the nakama-java client,  noticing that it was recently updated for gradle 6.5.1

I create a new jar (using `bash gradlew shadow`) from source, but when running my up
during DefaultClient construction, I now get a runtime error:

`Caused by: java.lang.IllegalArgumentException: cannot find a NameResolver for 127.0.0.1:7349 (Illegal character in scheme name at index 0: 127.0.0.1:7349)
` 
It is happening in DefaultClient constructor on builder.build() line (last line in below snippet)

```
       ManagedChannelBuilder builder = OkHttpChannelBuilder.forAddress(host, port).userAgent("nakama-java-client");
...
            this.managedChannel = builder.build();
```

I am not that familiar with grpc, but searching for it best I came up is this reference to the issue
https://stackoverflow.com/questions/62875526/grpc-java-cannot-find-a-nameresolver-when-using-in-osgi-bundle


I had tried upgrading to latest grpc packages (1.31.0 rather than what's in current build.gradle which is 1.21.0). I also upgraded to latest shadow Jar plugin (6.0.0)
None of that help.

At this point I am not sure if the problem is with my environment and when I used 2.0.4 version of the client, I simply was lucky.
Or, there is something wrong with my build of 2.0.5-snapshot.
Or, it is because I am not using Android java, and regular java might not be supported by design.

Logging this in case others might experience similar problem. Will post here if I find a resolution.

--- a/build.gradle
+++ b/build.gradle
@@ -262,6 +262,10 @@ task javadocJar(type: Jar) {
      }
  }
 
+shadowJar {
+    mergeServiceFiles()
+}
+
 task relocateShadowJar(type: ConfigureShadowRelocation) {
     target = tasks.shadowJar
     prefix = "nakama" // prefix all classpath entries with `nakama`

